draft-ietf-cdni-footprint-types-extensions.xml

```diff
@@ -67,8 +67,8 @@
             <t>
                 Open Caching architecture is a use case of Content Delivery Networks Interconnection (CDNI) in which the
                 commercial Content Delivery Network (CDN) is the upstream CDN (uCDN) and the ISP caching layer serves as the
-                downstream CDN (dCDN). The extensions specified in this document to
-                the Footprint and Capabilities Interface (FCI) are derived from requirements raised by Open Caching but
+                downstream CDN (dCDN). The document specifies extensions to
```

Comment:
@smishra1200 that looks good

Reply:
Updated Abstract section and broke out the Intro section